core/modules/wsched/ChunkState.h

```diff
@@ -32,6 +32,7 @@
 
 // System headers
 #include <algorithm>
+#include <chrono>
 #include <deque>
 #include <ostream>
 #include <set>
```

Comment:
`<algorithm>`, `<chrono>`, `<deque>` are not used in this header, can be removed, `<ostream>` can be replaced with `<iosfwd>`.